src/agb_eeprom.c

```diff
@@ -0,0 +1,273 @@
+
+#include "global.h"
+#include "gba/eeprom.h"
+
+
+typedef struct EEPROMConfig {
+    u32 unk_00;
+    u16 size;
+    u16 waitcnt;
+    u8 address_width;
+    // u8 filler[3];
+} EEPROMConfig;
+
+const char EEPROM_V124[] = "EEPROM_V124";
```

Comment:
The string is not linked properly. I think we should either just extern them or split data.s and link it properly.